.chloggen/fix_load-initial-servicemonitors.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: target allocator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix target allocator readiness check
```

Comment:
Does not not seem to be accurate

Reply:
This is from #2883, it'll disappear once that's merged. This PR is a bit messy in general right now, I published it to see if there's any fundamental objections to my approach here.